### DISCUSSION TITLE
iterative

### DISCUSSION THREAD

**Discussion by ttodosi** [DeabLabs/cannoli#42] [General]
hey checking in - still using cannoli all the time. any word on iterative features? i'd like to batch process, one by one a folder or dataview table.

**Comment by blindmansion** [comment]
Very soon! We've got a big update we're trying to release in hopefully less than two weeks which includes  a new group type called "parallel groups"

You can load parallel groups with a list of things, predefined or generated or retrieved from dataview, and each copy of the group will use a different item in that list when you reference that variable inside the group.

So stay tuned.

**Reply by ttodosi** [reply]
sounds interesting! looking forward to it
